.editorconfig

```diff
@@ -0,0 +1,8 @@
+[*]
```

Comment:
I was getting crazy with some weird tab ident in my IDE, so I've decided to add this